### DISCUSSION TITLE
Ecosystem Survey - Research Study

### DISCUSSION THREAD

**Discussion by 0x-allan** [akash-network/community#654] [Ideas]
Howdy! Back again with another research request. In support of the [App Migration Tool Prototype](https://github.com/orgs/akash-network/discussions/651) I shared in a separate discussion, I'm conducting a survey targeted towards Tenants or Users with the goal of validating certain value props and understanding general pain points. Tenants or users are the _ideal_ respondents, but other community members are also welcome to participate and share opinions, observations, or insights!

Link to the Google Form (Title: **Akash** Ecosystem Survey): https://github.com/orgs/akash-network/discussions/651

For added context, the results will inform my graduate-level service design project. No personally identifiable information will be collected, and all responses will be confidential! Any insights captured in this survey will be anonymized and/or aggregated in the ultimate submissions.

**Comment by ghost** [comment]
sounds like an exciting initiative to gather valuable insights for Akash 
Looking forward to see the results of your efforts

**Reply by 0x-allan** [reply]
thank you for the support @instafinanzas !

**Comment by rodri-r** [comment]
This does sound like a useful research. I'm also looking forward to reading the results. Thank you!

**Reply by 0x-allan** [reply]
appreciate the upvote! will share the results when I can
